File: Assets/PACKT_Scripts/bulletTemp.js
#pragma strict
var damage : int = 2;

function Start()
{
	Destroy(gameObject, 2);
}

function OnCollisionEnter(other : Collision)
{
	if(other.gameObject.tag == "Enemy")
	{
		other.gameObject.SendMessage("AddDamage", damage);
	}
}

File: Assets/PACKT_Scripts/bulletTemp.cs
using UnityEngine;
using System.Collections;

public class bulletTemp : MonoBehaviour {
	public int damage = 2;

	void Start () 
	{
		Destroy(gameObject, 2);
	}
	
	void OnCollisionEnter(Collision other)
	{
		if(other.gameObject.tag == "Enemy")
		{
			other.gameObject.SendMessage("AddDamage", damage);
		}
	}
}
